eppy/bunch_subclass.py

```diff
@@ -15,7 +15,7 @@
 
 import copy
 from eppy.EPlusInterfaceFunctions import readidf
-from bunch import Bunch
+from munch import Munch as Bunch
```

Comment:
Clever. Might be worth documenting why we've done this for the sake of future us.